src/ShowActivitySubHeader/index.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import LabelSmall from '../LabelSmall';
+import Wrapper from '../Wrapper';
+
+function ShowActivitySubHeader(props) {
+  return (
+    <div>
+      <Wrapper
+        size="side-bottom"
```

Comment:
You need a style here to give the component a white background as per the specs, otherwise on dark backgrounds the component may be lost. Click the dark green/blue box in the top right of storybook to preview the component with a dark background.